pipeline.ipynb

```diff
@@ -64,40 +73,56 @@
    },
    "outputs": [],
    "source": [
+    "#Set up Initial Basic Parameters#\n",
     "minian_path = \".\"\n",
-    "dpath = \"./demo_movies\"\n",
+    "dpath = \"./demo_movies/\"\n",
     "chunks = {\"frame\": 1000, \"height\": 50, \"width\": 50, \"unit_id\": 100}\n",
-    "subset = None\n",
+    "subset = dict(frame=slice(0,1000))\n",
```

Comment:
I don't think `slice(0,1000)` is a good default, but I do think it would be helpful to have a slice object that makes it clearer how to subset the data.  I have changed it to `subset = dict(frame=slice(0,None))` in commit `a6727ba`